views/components/contact-us.html

```diff
@@ -1,8 +1,12 @@
+{% if welshWebchatEnabled  %}
+<details class="govuk-details" open>
```

Comment:
I think this something they wanted regardless of the welsh webchat, worth confirming with Uma